modules/apps/frontend-theme/frontend-theme-classic/src/css/components/_navbars.scss

```diff
@@ -1,25 +1,10 @@
-.navbar-blank {
-	flex-wrap: wrap;
-
-	.lexicon-icon {
-		font-size: 0.625rem;
```

Comment:
I've added the class [`.lexicon-icon-sm`](https://github.com/liferay/clay/blob/master/packages/clay-css/src/scss/components/_icons.scss#L10) (0.5rem) to the icons inside the navbar-blank

Couldn't find the `size` API in the taglibs

- https://github.com/liferay/liferay-portal/blob/master/modules/apps/frontend-taglib/frontend-taglib-clay/src/main/java/com/liferay/frontend/taglib/clay/servlet/taglib/IconTag.java
- https://github.com/liferay/liferay-portal/blob/master/util-taglib/src/com/liferay/taglib/aui/IconTag.java